web/templates/user_group_settings/add_members_form.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="add_members_container">
+    <div class="pill-container person_picker">
+        <div class="input" contenteditable="true"
+          data-placeholder="{{t 'Add members. Use usergroup or #streamname to bulk add members.' }}">
+            {{~! Squash whitespace so that placeholder is displayed when empty. ~}}
+        </div>
```

Comment:
Added this as a new component instead of having if-else block to handle two cases in `add_subscribers_form` and we can also have more suitable classes here for groups UI.